tests/components/__snapshots__/storyshot.test.ts.snap

```diff
@@ -0,0 +1,8687 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
do we use this snapshot file somewhere? if not and it's just auto-generated, maybe we could remove it

Reply:
Yes, the storyshot add-on renders generates a jest snapshot. It should be committed so it will compare the future renders in pr with the snapshot as part of the testing. The snapshot file needs to be updated by running the storyshot command and committed every time there's an approved change in code implementation, so we might need a github action to do that. More information here: https://jestjs.io/docs/snapshot-testing